Cmdline/Action/Install.cs

```diff
@@ -67,7 +67,8 @@ public int RunCommand(CKAN.KSP ksp, object raw_options)
                     // Parse the JSON file.
                     try
                     {
-                        options.modules.Add(LoadCkanFromFile(ksp, filename).identifier);
+                        CkanModule m = LoadCkanFromFile(ksp, filename);
+                        options.modules.Add($"{m.identifier}={m.version}");
```

Comment:
Is this going to throw a Null ref if the .ckan file is incomplete?

Reply:
No; LoadCkanFromFile doesn't return null. If there's any problem with the file, it'll throw some other exception.

Reply:
Not an entire null, I mean if the .ckan file is missing a version

Reply:
The version field is required, so the JSON deserializer throws an exception inside LoadCkanFromFile if it's missing. I deleted the version from this file for testing:

```
$ mono ckan.exe install -c Astrogator-v0.7.8.ckan  --ksp steam
JSON deserialization error: Required property 'version' not found in JSON. Path '', line 67, position 1.: Required property 'version' not found in JSON. Path '', line 67, position 1.
Usage: ckan install [--with-suggests] [--with-all-suggests] [--no-recommends] [--headless] Mod [Mod2, ...]
```

Reply:
Cool. Thanks for checking that.